src/utils/aepp-wallet-communication/rpc/types.ts

```diff
@@ -1,3 +1,4 @@
+import { send } from '../../..'
```

Comment:
I would avoid importing from the `index.ts` to don't have looks like we have a cycle dependencies

Reply:
Good catch. Noted. Auto imported by the IDE it was.